packages/create-block/README.md

```diff
@@ -141,7 +141,7 @@ For example, running the `start` script from inside the generated folder (`npm s
 
 ## External Project Templates
 
-[Click here](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block-tutorial-template/) for information on External Project Templates
+[Click here](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/packages-create-block-external-template/) for information on External Project Templates
```

Comment:
It doesn't look relevant based on the PR's description, but it's a good change we can include.

Reply:
A I seem to have goofed this simple thing... how do I remove that file?

Reply:
Curious. This change was already merged in #56106, but it's not harmful to do it again